tests/tests_files_files.yml

```diff
@@ -97,7 +103,8 @@
       failed_when: __result.stat.exists
 
     - name: Grep output to messages line
-      command: /bin/grep "{{ __default_system_log }}" {{ __test_outputfiles_conf }}
+      command: "/bin/grep
```

Comment:
with `command` it is better to use `>-` which will generate a single line string with each newline replaced with a single space